pyiron-atomistic/Dockerfile

```diff
@@ -1,45 +1,11 @@
-ARG ROOT_CONTAINER=jupyter/base-notebook:36d857bb0121
```

Comment:
In contrast when linking to an external repository I prefer to use the tag rather than latest, so in case the jupyter notebooks container changes and breaks the compatibility to the pyiron container our images do not fail. This should be a manual step as it requires testing. Ideally we should have a bot which checks for new releases of the jupyter notebook image and opens a pull request every time a new image is released so we can confirm it is still working with our image.

Reply:
In the cases of using pyiron images, I used the latest tag but in the case of jupyter/base-notebook I used a certain tag from one of their recent tags.
It is a great idea to have a bot to update the tag for jupyter-base. I will look it up.

Reply:
Sorry my mistake - here it is fine to use `pyiron/base:latest `. The bot is the next level first we should get the current stack working then we can think about the bot.